File: CodeForAndroid/DialogTest/app/src/main/java/com/example/dialogtest/MainActivity.java
package com.example.dialogtest;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    private Button ShowButton;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        ShowButton = findViewById(R.id.ShowButton1);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    public void ShowDialog(){
        AlertDialog dialog = null;

    }
}

File: CodeForAndroid/KotlinTest2/app/src/main/java/com/example/kotlintest2/MainActivity.kt
package com.example.kotlintest2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {

    var button:Button = findViewById(R.id.bottom)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }
}